includes/reader-activation/class-reader-activation.php

```diff
@@ -1638,7 +1638,7 @@ public static function register_reader( $email, $display_name = '', $authenticat
 				 * Create WooCommerce Customer if possible.
 				 * Email notification for WooCommerce is handled by the plugin.
 				 */
-				$user_id = \wc_create_new_customer( $email, $user_data['user_login'], $user_data['random_password'], $user_data );
+				$user_id = \wc_create_new_customer( $email, $user_data['user_login'], $user_data['user_pass'], $user_data );
```

Comment:
Is this change intended? I can't see how it relates to PR's description.

Reply:
It was an initial attempt at addressing the fatal described in the description, but turned out to be a red herring. However, I think it's still not a bad practice to use the `\` namespace for WP globals since the file itself is also namespaced. Just for code readability/clarity.

Reply:
Whoops, looked at the wrong line. This change is intended, yes. I'll update the testing instructions